### ISSUE TITLE
Null warning not reported for constant member

### ISSUE THREAD

**Issue by cston** [dotnet/roslyn#23698]
`const string A = null;` should generate `warning: cannot convert null to non-nullable reference`:
```
class C
{
    const string A = null;
}
```

**Comment by jaredpar** [comment]
@chsienki this is likely similar to the attribute work that you did for enabling nullable analysis.

**Comment by jaredpar** [comment]
Looks like this is a duplicate of #29995
